### PR TITLE
Agreement List expanded area updated with Budget Lines section

### PR DESCRIPTION
**Pull request by stevtek**
## What changed

Expanded area in the Agreements list had been updated with new UX
* Budget Lines section has beed added with counts by status
* Notes have been replaced with Description

## Issue

#1655 #965 

## How to test

View the agreements list and expand a row

## Screenshots

image

## Definition of Done Checklist
- [x] OESA: Code refactored for clarity
- [x] OESA: Dependency rules followed
- [x] Automated unit tests updated and passed
- [x] Automated integration tests updated and passed
- [x] Automated quality tests updated and passed
- [x] Automated load tests updated and passed
- [x] Automated a11y tests updated and passed
- [x] Automated security tests updated and passed
- [x] 90%+ Code coverage achieved
- [x] Form validations updated

## Links

_If relevant, e.g. for a link to a piece of markdown documentation_

### CODE REVIEW

**Review by tdonaworth — APPROVED** [review]

File: frontend/src/components/Agreements/AgreementsTable/AgreementTableRow.jsx [hunk 1]
```diff
@@ -149,6 +159,41 @@ export const AgreementTableRow = ({ agreement }) => {
...
+    const StatusCountItem = ({ status, count }) => {
+        const label = convertCodeForDisplay("budgetLineStatus", status);
+        return (
+            <div className="display-flex flex-justify margin-top-1">
+                <div className="">
+                    <div className="display-flex flex-align-center">
+                        <FontAwesomeIcon
+                            icon={faCircle}
+                            className={`${statusToClassName(status)} height-1 width-1 margin-right-05`}
+                            // style={{ verticalAlign: "middle" }}
```
Comment (johndeange) [inline frontend/src/components/Agreements/AgreementsTable/AgreementTableRow.jsx:179]: please remove commented code

File: frontend/src/components/Agreements/AgreementsTable/AgreementTableRow.jsx [hunk 2]
```diff
@@ -167,16 +212,36 @@ export const AgreementTableRow = ({ agreement }) => {
...
+                        className="margin-0"
+                        style={{ maxWidth: "400px" }}
+                    >
+                        {agreementDescription ? agreementDescription : "No description created."}
+                    </dd>
+                </dl>
                 <dl
                     className="font-12px"
-                    style={{ marginLeft: "9.0625rem" }}
+                    style={{ marginLeft: "50px" }}
```
Comment (johndeange) [inline frontend/src/components/Agreements/AgreementsTable/AgreementTableRow.jsx:229]: 50 px = 3.125 rem if you want to keep the units

**Review by johndeange — APPROVED** [review]

File: frontend/src/components/Agreements/AgreementsTable/AgreementTableRow.jsx (hunk 1 above)
Comment (stevtek) [reply frontend/src/components/Agreements/AgreementsTable/AgreementTableRow.jsx]: done

File: frontend/src/components/Agreements/AgreementsTable/AgreementTableRow.jsx (hunk 2 above)
Comment (stevtek) [reply frontend/src/components/Agreements/AgreementsTable/AgreementTableRow.jsx]: done

**Pull request merged**